### PR TITLE
Email contact tokens URL mapping

### PR DESCRIPTION
**Pull request by mqueme**
[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | y
| New feature? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #2571
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
When sending an email to a segment with more than one contact, if we have a contact field token with a link, it gets replaced with the first encounter of the link. This PR fixes it by not replacing URL with a tracking link. In theory Mautic will not track contact URL's since each contact url can be different.
[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. add a custom field or use an existing URL field
2. add a contact field token in an email, and send it, this will get replaced by the first encounter of the URL token, and all emails will end up with that same trackable link.

#### Steps to test this PR:
1. Apply PR, follow steps above
2. contact's URLs will be displayed with each contact's respective URL.

### CODE REVIEW

File: app/bundles/PageBundle/Model/TrackableModel.php [hunk 1]
```diff
@@ -245,11 +245,8 @@ public function parseContentForTrackables($content, array $contentTokens = [], $
             'first_pass' => [
                 // Remove internal attributes
                 // Editor may convert to HTML4
-                'mautic:disable-tracking=""' => '',
-                // HTML5
-                'mautic:disable-tracking' => '',
+                'data-mautic-tracking="disable"' => '',
```
Comment (escopecz) [inline app/bundles/PageBundle/Model/TrackableModel.php]: Maybe we should keep the previous attributes for BC. Some users use it (https://github.com/mautic/mautic/issues/2927#issuecomment-293379553)

File: app/bundles/LeadBundle/EventListener/EmailSubscriber.php [hunk 2]
```diff
@@ -82,7 +82,7 @@ public function onEmailGenerate(EmailSendEvent $event)
         $content .= $event->getContent();
         $content .= $event->getPlainText();
         $lead = $event->getLead();
-
+       // $this->factory->getLogger()->addError(print_r($content, true));
```
Comment (escopecz) [inline app/bundles/LeadBundle/EventListener/EmailSubscriber.php]: This should be probably removed.

**Comment by escopecz** [comment]
I'm moving this out of 2.8.0. We'll have to discuss with @alanhartless what the $doNotTrack values should be.

**Comment by LAN-Laurene** [comment]
@mqueme 
Hi,
Is this PR usable? We are suffering from the same problem. I see that the tests are failing. Is there a real problem or is it a false alert? 
Thanks,

**Comment by mqueme** [comment]
I think it may be a false alert, because these contact fields should be returning the token and not the URL, as the Test expects. Please test it and get back to us with your results. Thanks!

**Comment by npracht** [comment]
Hello,

We have tested the PR and it works correctly for us. We didn't see any issues anywhere else.

N.

**Comment by dongilbert** [comment]
Closing in favor of #4028

**Pull request closed without merging**